Automattic/wp-calypso
# My Home: Define card identifiers in constants file

Requires D43588-code.

#### Changes proposed in this Pull Request

In order to make the cards UIDs less fragile, this PR adds a constants file defining all UIDs and aiming to be the single source of truth for the mapping between client and server.

#### Testing instructions

- Apply D43588-code and sandbox the API.
- Go through the site setup journey (create a site, complete site setup, launch, view first task, ...) and make sure there are no regressions (behavior should match production).

Fixes #41758
